apidump/net.go

```diff
@@ -29,6 +31,40 @@ func (w interfaceWrapper) Addrs() ([]net.Addr, error) {
 	return w.addrs, nil
 }
 
+// Show a warning about failure to check permission, and
+// return an appropriate top-level error message.
+func showPermissionErrors(sampleError error) error {
+	// TODO: pcapPermError needs an Unwrap method to use error.Is().  But
+	// I don't know what error this is anyway! So we're back to substring search.
+	if strings.Contains(sampleError.Error(), "Operation not permitted") {
+		// Permission denied == not enough capabilities
+		// Are we running as root?
+		if os.Geteuid() == 0 {
+			if env.InDocker() {
+				printer.Warningf("Although you are running as root, this container lacks the CAP_NET_RAW capability.\n")
+				printer.Warningf("It might be that you are in a PaaS which disallows packet capture, or the local configuration has disabled that privilege by default.\n")
```

Comment:
Not all PaaS's disallow packet capture.
```suggestion
				printer.Warningf("It might be that you are in a PaaS that disallows packet capture, or the local configuration has disabled that privilege by default.\n")
```